hat/assets/js/apps/Iaso/domains/dataSources/components/VersionsDialog.js

```diff
@@ -268,8 +274,7 @@ const VersionsDialog = ({ renderTrigger, source }) => {
         >
             <Table
                 data={sortedData}
-                columns={tableColumns(source)}
-                redirectTo={() => {}}
+                columns={tableColumns(source, forceRefreshParent)}
```

Comment:
This is ok for now 😉  Next time you can use a hook to get columns then it will be memoized